src/app/core/internal/view/rendering/prims/ArcCircle.ts

```diff
@@ -0,0 +1,41 @@
+import {Vector} from "Vector";
+
+import {Circle} from "./Circle";
+
+
+/**
+ * A representation of an ArcCircle shape.
+ */
+export class ArcCircle extends Circle {
+    protected a0 = 0;
+    protected a1 = 2*Math.PI;
+
+    /**
+     * Constructor for ArcCircle.
+     *
+     * @param pos    The position.
```

Comment:
I know this is probably copied and pasted over from master, but this `@param` description should be a little more descriptive or just nonexistent. I assume is it the position of where the center of the circle would be if it a full circle was drawn, but it could be misinterpreted as the position at which to start drawing from.

Reply:
👺